specification/devtestlabs/resource-manager/readme.ruby.md

```diff
@@ -24,3 +25,13 @@ Please also specify `--ruby-sdks-folder=<path to the root directory of your azur
 namespace: "Azure::DevTestLabs::Mgmt::V2016_05_15"
 output-folder: $(ruby-sdks-folder)/management/azure_mgmt_devtestlabs/lib
 ```
+
+### Tag: package-2018-09 and ruby
```

Comment:
This change is new from the already reviewed and approved PR. I was not sure if I had to add it or not. Let me know if it should be reverted.